awx_collection/plugins/modules/project.py

```diff
@@ -169,12 +169,6 @@
       required: False
       default: 2
       type: float
-    signature_validation:
-      description:
-        - Whether to validate the signature during project update.
-      required: False
-      default: False
-      type: bool
     signature_validation_credential:
```

Comment:
description need to be changed 

no longer have a IF condition and also setting this will imply enabling signature validation